app/Http/Controllers/ProjectController.php

```diff
@@ -2,20 +2,26 @@
 
 namespace Pibbble\Http\Controllers;
 
+use Auth;
 use Pibbble\Project;
 use Illuminate\Http\Request;
 use Pibbble\Http\Requests;
 use Pibbble\Http\Controllers\Controller;
+use Cloudinary\Uploader;
```

Comment:
I am sending it right away.


Best Regards,

*Surajudeen Akande*
Software Developer, *Andela*
+2348037690966 | Skype: sirolad <https://webapp.wisestamp.com/#>
<http://facebook.com/runthinz>  <http://twitter.com/babatoclean1>
<http://github.com/andela-sakande>

On Mon, Nov 9, 2015 at 11:16 AM, Prosper Otemuyiwa <notifications@github.com
> wrote:

> In app/Http/Controllers/ProjectController.php
> <https://github.com/andela/pibbble/pull/105#discussion_r44258618>:
>
> >  use Pibbble\Project;
> >  use Illuminate\Http\Request;
> >  use Pibbble\Http\Requests;
> >  use Pibbble\Http\Controllers\Controller;
> > +use Cloudinary\Uploader;
>
> Where's the PR for that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela/pibbble/pull/105/files#r44258618>.
>